react/client/src/components/PageContainer.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components'
+
+import Header from 'components/Header'
+import Form from 'components/Form'
+import FormHeader from 'components/FormHeader'
+import Landing from 'components/pages/Landing'
+
+// import device from 'styles/breakpoints'
+
+interface StyleProps {
+  theme?: string,
+  background?: string;
+}
+
+const basicStyles = `
+  background: white;
+  color: #131313;
+`
+
+const purpleStyles = `
+  background: #9903ff;
+  color: white;
+`
+
+const StyledContainer = styled.div<StyleProps>`
+  background: ${props => props.background};
+  color:white;
+  padding: 18px;
+  display: flex;
+  flex: 1 0 auto;
+  flex-direction: column;
+
+  ${props => {
+    switch (props.theme) {
+      case 'purple':
+        return purpleStyles;
+
+      case 'basic':
+      default:
+        return basicStyles;
+    }
+  }}
+`
+
+interface PageProps {
+  history: {
+    location: {
+      pathname: string
+    },
+    push: (address: string) => void
+  },
+  match: {
+    params: {
+      page: string
+    }
+  }
+};
+
+const PageContainer: React.FC<PageProps> = ({ history, match }) => {
```

Comment:
`PageContainer.tsx` replaced `PersonalStatementContainer.tsx` for clarity. I'm a new developer coming in and expected "PersonalStatementContainer" to be for something specific when it is actually a high level wrapper. I think we could do with a smidge of architectural change but for now I've simplified it to be the general-purpose layout.